MoviesMVP/MoviesMVPTests/RealmServiceTests.swift

```diff
@@ -0,0 +1,35 @@
+// RealmServiceTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import XCTest
+
+@testable import MoviesMVP
+
+/// Realm service tests
+final class RealmServiceTests: XCTestCase {
+    // MARK: - Private properties
+
+    private var realmService: RealmServiceProtocol?
+  
+    // MARK: - Public methods
+
+    override func setUp() {
+        realmService = RealmService()
+    }
+
+    override func tearDown() {
+        realmService = nil
+    }
+
+    func testRealmService() {
```

Comment:
```suggestion
    func testRealmService() {
```
проверку чтения и записи разнеси в разные методы